AustLII and NZLII.js

```diff
@@ -113,6 +116,8 @@ function scrape(doc, url) {
 			var voliss = text(doc, 'head>title');
 			// e.g. C & M [2006] FamCA 212 (20 January 2006)
 			newItem.caseName = voliss.replace(/\s?\[.*$/, '');
+			newItem.title = newItem.caseName;
+			
```

Comment:
Why did you change that? For a `case` the title is actually the `caseName`, thus the title here will be deleted before saving into the database. Please change back these two lines.

Reply:
Because without a `.title`, the Zotero connector in Chrome fails with an error, and the new item doesn't get created in Zotero.

I agree it shouldn't be necessary, but right now it is.

Reply:
Uff, yes indeed. There is currently an error for cases also with Firefox, e.g. http://www8.austlii.edu.au/cgi-bin/viewdoc/au/cases/cth/AICmr/2017/134.html @dstillman Is it intended that a missing `title` but present `caseName` leads to an error?